sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/src/Customizations/Models/ConversionErrorCode.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core;
+
+namespace Azure.MixedReality.ObjectAnchors.Conversion.Models
+{
+    /// <summary> The ConversionErrorCode. </summary>
```

Comment:
If we need this custom model we should update the docs.

Reply:
Removed through swagger change, but docs do need updating